src/components/Web3Modals/EnsureReadOnlyConnection/index.tsx

```diff
@@ -3,14 +3,12 @@ import { useEffect, useMemo } from 'react';
 import { useAccount, useConnect, useNetwork } from 'wagmi';
 
 const EnsureReadOnlyConnection = () => {
-  const { chains } = useNetwork();
-  const defaultChain = useMemo(() => {
+  const { chains, chain } = useNetwork();
+  const validChain = useMemo(() => {
     if (chains?.length === 0) return null;

```

Comment:
I think you can remove this line, right?

Reply:
That's a very good question.

I tinkered with it to get an answer, and it seems that its removal won't affect the connection.

Maybe @mprasanjith can chime in?